src/style.scss

```diff
@@ -67,36 +65,46 @@
     font-family: 'Penumbra Web Pro';
     font-style: normal;
     font-weight: 400;
-    src: url('../../fonts/penumbra-web-pro/PenumbraWebPro-Serif.woff2')
-        format('woff2');
+    src: url('assets/fonts/penumbra-web-pro/PenumbraWebPro-Serif.woff')
+        format('woff');
 }
 
 @font-face {
     font-family: 'Laski Sans';
     font-style: normal;
     font-weight: 400;
-    src: url('../../fonts/laski-sans/Laski-Sans-Regular.woff2') format('woff2');
+    src: url('assets/fonts/laski-sans/Laski-Sans-Regular.woff2') format('woff2');
 }
 
 @font-face {
     font-family: 'Laski Sans';
     font-style: normal;
     font-weight: 600;
-    src: url('../../fonts/laski-sans/Laski-Sans-SemiBold.woff2') format('woff2');
+    src: url('assets/fonts/laski-sans/Laski-Sans-SemiBold.woff2')
+        format('woff2');
 }
 
 @font-face {
     font-family: 'Laski Sans';
     font-style: normal;
     font-weight: 700;
-    src: url('../../fonts/laski-sans/Laski-Sans-Bold.woff2') format('woff2');
+    src: url('assets/fonts/laski-sans/Laski-Sans-Bold.woff2') format('woff2');
 }
 
 @font-face {
     font-family: 'cosmere-dingbats';
     src: url('https://dl.dropboxusercontent.com/scl/fi/9909gen4fd0oveyzfposx/CosmereDingbats-Regular.otf?rlkey=ig6odq9hxyo1st8kt3ujp1czz&st=72qrads3&raw=1');
 }
 
+.application h1,
+span.document-name {
+    font-family: 'Penumbra Web Pro';
```

Comment:
Small issue with this; It's hard to tell what letters are capitalised in document names. I think we can just add: `font-variant: small-caps;` as a fix.

Reply:
I did try and play around with this myself (bit of a sucker for small caps!). It looks good, but then I realised that the books just use the basic all caps approach so was keeping in line with that?

Reply:
I think we should use the small caps option here. As when editing the name it's impossible to see what letters you've got capitalised, but it does show in the items list on the side panel. It just kind of feels bad from a UX perspective.

Reply:
#74 raised for further discussion, have set the variant on the branch for now